docs/content/connectors/oracle-cdc.md

```diff
@@ -169,6 +169,7 @@ Overall, the steps for configuring CDB database is quite similar to non-CDB data
      GRANT SELECT ANY TRANSACTION TO flinkuser CONTAINER=ALL;
      GRANT LOGMINING TO flinkuser CONTAINER=ALL;
      GRANT CREATE TABLE TO flinkuser CONTAINER=ALL;
+     -- need not to execute if set scan.incremental.snapshot.enabled=true(default)
      GRANT LOCK ANY TABLE TO flinkuser CONTAINER=ALL;
```

Comment:
Since it is enabled by default, does it mean that the commands here can be annotated?